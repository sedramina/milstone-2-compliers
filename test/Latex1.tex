\documentclass{article}
\usepackage[utf8]{inputenc}



\begin{document}
\title{First Template}
\date{March 2016}
\maketitle


\section{Item List}
In this template you have an itemized list that you have to generate the tokens for using the lexical analyzer implemented in this milestone. \\

\begin{itemize}
\item item 1
\item item 2
\item item 3
\end{itemize}
%This line is commented and it should not be shown in your output

\section{Math Mode}
In this section you have the math mode of latex presented using the dollar sign. In this mode you can present any form of mathematical expression. However, for the scope of the the project we only restricted it to numbers, variables(might be character, or a character followed by a number as shown below), operators(+,-,*,/,=) and functions. \\

$5$
\\
$3x$
\\
$5+z$
\\
$f_1$

\section{New Line}
The new Line in latex in expressed using double backslash, which will be also be the case in the project as well. So you might need to change your previous lexer a bit to adapt to this.



\end{document}