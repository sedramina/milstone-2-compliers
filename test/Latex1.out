First Template
March 2016
Item List
In this template you have an itemized list that you have to generate the tokensfor using the lexical analyzer implemented in this milestone.
item 1
item 2
item 3
Math Mode
In this section you have the math mode of latex presented using the dollarsign. In this mode you can present any form of mathematical expression. How-ever, for the scope of the the project we only restricted it to numbers, vari-ables(might be character, or a character followed by a number as shown below),operators(+,-,*,/,=) and functions.

5

3x

5+z

f1
New Line
The new Line in latex in expressed using double backslash, which will be alsobe the case in the project as well. So you might need to change your previouslexer a bit to adapt to this.
